### ISSUE TITLE
only export keyframes for node animations

### ISSUE THREAD

**Issue by paroj** [OGRECave/blender2ogre#87]
> This function processess the objects animation frame by frame, insted of using its keyframes

Im not fully happy with this. While I understand the need of it, there _are_ serious drawbacks.
- what are the FPS of the animation? 24 as the blender default? This might or might not do what the user expects.
- by getting rid of the keyframes, there are no keyframes to easily modify the animation in code
- unrolling the animation in `.skeleton` is less evil as that is a binary format not edited by humans. `.scenes` on the other hand seem to be designed to be human readable. But with unrolling animations you end with something like this:
![vrar20-3dvis](https://user-images.githubusercontent.com/4941802/115129312-07ac6d80-9fe5-11eb-95dd-7b2b5c445b1c.png)

I think in this case, I would just for once advocate for a user-switch (default-off):
-  [ ] only write node keyframes

(we can also merge this as is and add that later)

_Originally posted by @paroj in https://github.com/OGRECave/blender2ogre/issues/86#issuecomment-821902071_

**Comment by oldmanauz** [comment]
The code for this feature exists, it's commented out though.
Was it disabled for a reason? 
Do you want it re-enabled?

**Comment by paroj** [comment]
> Was it disabled for a reason?

yes, see: https://github.com/OGRECave/blender2ogre/pull/86#issuecomment-822646691

**Comment by sercero** [comment]
@oldmanauz I was the one who implemented this feature.

I was unable to come up with a solution on how to export the keyframes, it didn't seem as simple as exporting every frame.

**Comment by oldmanauz** [comment]
@sercero I see your logic. Is it better to have a well documented part-implementation that might suit some than potentially never having a solution?

**Comment by paroj** [comment]
I would prefer having that part implementation

**Comment by sercero** [comment]
@oldmanauz, @paroj: of course I have no problem with more options, mostly my concern is that the option to export every keyframe is still present.

Thanks!
